models/baseModels/Invoice/Invoice.ts

```diff
@@ -164,14 +167,19 @@ export abstract class Invoice extends Transactional {
 
   async afterSubmit() {
     await super.afterSubmit();
-
     // update outstanding amounts
+    if (this.schemaName === ModelNameEnum.SalesQuote) {
+      return;
+    }
```

Comment:
place these lines above the comment so as not to be confused with it is related to updating the outstanding amounts

Reply:
leave a line after the end of if condition.